dash/frontend/src/app/modules/private/pages/image/image-scan-result/image-scan-result.component.ts

```diff
@@ -351,6 +351,10 @@ export class ImageScanResultComponent implements OnInit, AfterViewInit, OnDestro
       });
   }
 
+  downloadReport() {
```

Comment:
Is there a reason to have this function over calling print directly?

Reply:
Yeah, Angular doesn't believe in straight prints lol
![image](https://github.com/m9sweeper/m9sweeper/assets/50845926/af446884-eaf5-4b45-a2b4-0739703b99bb)